tests/distributions/test_bivariate_von_mises.py

```diff
@@ -0,0 +1,122 @@
+# Copyright Contributors to the Pyro project.
```

Comment:
Could you also add a standard test case to tests/distributions/conftest.py?  These tests check for standard things like shape errors and correct metadata.  You may need to add special case handling logic in a few places like test_gof; let me know if you'd like any help working through test errors.

Reply:
I'll give it a shot and ping you if I can't figure out the cause myself.